packages/nepal/nepal.svg

```diff
@@ -0,0 +1,536 @@
+		c-7.458-2.615-11.157,2.335-15.099,7.002c-2.543,3.012-4.945,6.144-8.403,8.262c-1.016,0.622-2.027,1.907-0.776,3.078
+		c0.545,0.51,2.209,0.678,2.716,0.244c6.186-5.308,12.932-2.315,19.494-1.408c2.795,0.386,5.086,1.425,6.095-2.883
+		c1.098-4.684,5.399-4.927,9.146-3.721c7.959,2.561,16.032,5.137,21.23,12.492c0.867,1.227,1.666,3.192,2.794,3.434
+		c7.05,1.515,12.763,3.321,6.896,12.214c-0.597,0.905,1.913,0.846,2.977,1.211c2.457,0.845,4.693,3.031,3.21,4.948
+		c-3.9,5.04,0.622,8.855,1.426,12.942c1.005,5.104,8.548,6.879,14.822,4.351c0.864-0.348,2.102-1.875,2.157-1.82
+		c5.627,5.486,13.095-1.907,15.462-0.21c8.206,5.885,14.494,1.126,21.528-0.726c0.875-0.23,2.039-0.129,2.847,0.269
+		c6.826,3.365,11.709,1.012,13.425-6.794c0.506-2.3,0.672-4.553,3.397-5.263c3.032-0.79,3.312,2.544,4.818,3.325
+		c5.564,2.885,10.107,7.104,15.297,10.41c3.544,2.258,5.925,10.009,2.536,11.887c-8.019,4.444-5.428,12.068-4.809,17.474
+		c0.879,7.675-3.109,10.593-8.085,13.079c-4.056,2.026-5.88,4.4-5.204,8.764c0.001,0.001,0.001,0.001,0.001,0.002
+		c10.287,0.914,19.459-0.775,24.704-11.624c3.776,7.31,3.492,14.538,4.767,21.416c2.122,11.449,15.176,20.354,26.551,19.078
+		c4.463-0.501,11.959-4.68,13.085,5.272c0.286,2.53,3.069,3.167,5.352,3.459c3.576,0.458,6.702,1.779,9.81,3.637
+		c5.888,3.52,10.606,1.144,10.763-5.747c0.155-6.757,4.267-9.121,9.645-10.671c1.674-0.483,3.703-0.194,5.124-1.022
+		c8.897-5.186,15.976-1.886,23.334,3.734c4.252,3.247,7.973,10.066,14.487,7.133c5.318-2.395,12.242-1.879,15.849-8.065
+		c1.663-2.852,3.728-3.573,7.068-0.558c5.636,5.088,11.884,3.701,15.663-2.84c2.282-3.949,3.458-8.351,9.326-3.527
+		c2.722,2.237,6.492,1.051,8.547-2.168c2.854-4.469,4.989-2.499,7.153,0.473c3.358,4.611,6.545,9.402,12.061,11.78
+		c2.85,1.228,5.008,2.817,6.262,5.988c1.516,3.831,5.154,4.308,8.922,3.998c3.23-0.267,5.182-1.575,5.227-5.108
+		c0.029-2.256,1.824-3.275,3.364-4.653c7.891-7.065,16.009-13.482,11.603-26.837c-1.007-3.053,0.297-10.222,2.392-15.439
+		c4.093-10.189,11.545-18.222,13.966-29.974c2.728-13.237,4.202-25.51,0.535-38.31c-1.192-4.161,1.921-9.34-1.705-12.352
+		c-5.888-4.891-3.544-13.234-8.718-18.203c-1.127-1.082-0.852-2.961-0.188-4.328c2.177-4.474,2-7.592-3.722-8.994
+		c-1.667-0.408-3.462-2.366-4.209-4.052c-2.059-4.646-3.945-8.346-10.404-7.236c-2.896,0.498-4.02-3.335-4.629-6.087
+		c-0.428-1.937,0.177-5.009-0.915-5.747c-7.508-5.077-3.148-10.593-1.018-16.031c0.537-1.37,1.485-2.449,1.215-4.178
+		c-2.025-12.96,3.045-24.515,8.024-35.83c2.008-4.563,2.297-7.951-0.444-11.835c-3.359-4.76-2.457-9.076,2.235-12.064
+		c3.313-2.11,3.833-4.244,2.202-7.5c-3.287-6.559-4.859-11.848-2.33-20.578c3.388-11.691,11.561-20.04,15.435-30.953
+		c1.466-4.13,4.344-7.852,4.318-12.506c-0.016-2.871,0.515-5.477,1.99-7.917c3.865-6.394,9.613-12.254,5.717-20.834
+		c-0.322-0.709,0.04-2.115,0.591-2.772C1865.543,684.296,1861.545,681.561,1856.507,678.228z"/>
+	<path 
+		id="border"
+		name="Provincial and territorial borders"
```

Comment:
The SVG must contain only the provinces listed in the documentation and the borders should be defined by the other `<path>` elements.